### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by qsymmachus**
Adds some basic integration tests for our controllers.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: ruby
```
Comment (qsymmachus) [inline .travis.yml]: Configuration file for our Travis CI job

File: config/database.travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+test:
```
Comment (qsymmachus) [inline config/database.travis.yml]: Test DB configuration specific to Travis CI, this overrides our default test DB config when running on Travis.

File: test/controllers/events_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class EventsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @events_uri = "/events"
+  end
+
+  test "index" do
```
Comment (dominic) [inline test/controllers/events_controller_test.rb]: How does this get around the lack of NationBuilder credentials?

Comment (qsymmachus) [reply test/controllers/events_controller_test.rb]: I've stored the credentials as environment variables in Travis (don't worry, they're behind auth and encrypted)

**Pull request merged**